src/lib/transformers.ts

```diff
@@ -25,9 +25,10 @@ export function filterer<T>(
 	};
 }
 
-export type BatcherOptions = {
+export type BatcherOptions<T> = {
 	minBatchSize?: number;
 	flushTimeout?: number;
+	errorHandler?: (e: Error, batch: T[]) => void;
```

Comment:
This option needs documentation. Especially that the default behavior when it is missing, is to just throw the error, such that the original write or end call is rejected with it.

Reply:
Agreed. Will do...